airflow/cli/commands/webserver_command.py

```diff
@@ -137,7 +137,9 @@ def ready_prefix_on_cmdline(proc):
             try:
                 cmdline = proc.cmdline()
                 if cmdline:
-                    return settings.GUNICORN_WORKER_READY_PREFIX in cmdline[0]
+                    return cmdline[0].contains(
+                        (settings.GUNICORN_WORKER_READY_PREFIX,)
```

Comment:
this logic does not work for internal API cus it has a different string; if you turn on debug logging then you can see that it thinks workers are not ready when they are